src/block_cache/block_chunkcache.c

```diff
@@ -881,9 +881,12 @@ __wt_chunkcache_get(WT_SESSION_IMPL *session, WT_BLOCK *block, uint32_t objectid
                  * Increment the access count for eviction. If we are accessing the new chunk, the
                  * access count would have been incremented on it when it was newly inserted to
                  * avoid eviction before the chunk is accessed. So we are giving two access counts
-                 * to newly inserted chunks.
+                 * to newly inserted chunks. Additionally, cap the access count to optimize the
+                 * eviction process. This capping helps in particularly focusing on evicting older
```

Comment:
NP:  "in particularly" -> "particularly in"

Reply:
Thanks, fixed in [38f6e13](https://github.com/wiredtiger/wiredtiger/pull/9920/commits/38f6e13cbef32220e0a3d48270fb3d2f519f01b1)